### ISSUE TITLE
Separate National Society data from Country data

### ISSUE THREAD

**Issue by tovari** [IFRCGo/go-api#1618]
it might make sense to separate Country and NS data in GO.
The sources of data are different, the admins, who might modify the data are different as well.

There is a data source for NS data, which is kept maintained and contains much more information about the NS than we have at the moment. It would ease the NS data maintenance efforts.
https://go-api.ifrc.org/api/NationalSocieties
(same api we use for the appeal import)

Data sample:
{
        "DON_code": "DBA001",
        "NSO_DON_name": "The Red Cross Society of Bosnia and Herzegovina",
        "NSO_LNG_LCID": "1033",
        "NSO_CUR_code": "BAM",
        "NSO_CUR_name": "Bosnia Herzegovina Convertible Mark",
        "NSO_GST_code": "BA",
        "NSO_GST_name": "Bosnia and Herzegovina",
        "NSO_ZON_code": "SP5",
        "NSO_ZON_name": "Europe and Central Asia",
        "NSO_ZON_URL": "europe",
        "NSO_FIPS_code": "BK",
        "NSO_recognizedByFederation": "True",
        "NSO_countryCapital": "Sarajevo",
        "NSO_forPublicView": "True",
        "ADD_DON_Code": "DBA001",
        "ADD_notes_ID": "500C2429624F7372C1256B01005CE370",
        "ADD_name": "Bosnia and Herzegovina - The Red Cross Society of Bosnia and Herzegovina",
        "ADD_nameOriginal": "Red Cross Society of Bosnia and Herzegovina",
        "ADD_country_code": "BA",
        "ADD_webSite": "http://www.rcsbh.org/",
        "ADD_address1": "Kranjceviceva 2",
        "ADD_address2": null,
        "ADD_address3": null,
        "ADD_postalAddress1": null,
        "ADD_postalAddress2": null,
        "ADD_postalAddress3": null,
        "ADD_city_code": "Sarajevo",
        "ADD_postalCity": null,
        "ADD_zipCodeBefore": "71000",
        "ADD_zipCodeAfter": null,
        "ADD_postalZipCodeBefore": null,
        "ADD_postalZipCodeAfter": null,
        "ADD_phone": "(387) 33 263 935",
        "ADD_fax": "(387) 33 263 935",
        "ADD_telex": null,
        "ADD_telegram": null,
        "ADD_email": "rcsbh@rcsbh.org",
        "ADD_language": "English",
        "ADD_orgCreation": "1914 and independent 15.12.2000",
        "ADD_recognisedByFederation": "08.05.2001",
        "ADD_admittedByFederation": "07.11.2001",
        "ADD_lastStatutes": null,
        "ADD_recognisedByGovernment": "07.09.2000",
        "ADD_lastUpdated": "3/24/2022 9:26:16 AM",
        "ADD_zone": "Europe and Central Asia",
        "ADD_Id": "1706",
        "ADD_facebook": "https://www.facebook.com/dckbih",
        "ADD_twitter": "https://twitter.com/RedCrossBH",
        "ADD_mailingLanguageEnglish": "True",
        "ADD_mailingLanguageFrench": "False",
        "ADD_mailingLanguageSpanish": "False",
        "ADD_mailingLanguageArabic": "False",
        "ADD_mailingLanguageRussian": "False",
        "ADD_mailingLanguageChinese": "False",
        "ADD_swissPostalZone": "2",
        "ADD_geographicalZone": "4",
        "NSN_notes_ID": "500C2429624F7372C1256B01005CE370",
        "NSN_nameWorking": "Bosnia and Herzegovina - The Red Cross Society of Bosnia and Herzegovina                                                                              ",
        "NSN_nameEnglish": "The Red Cross Society of Bosnia and Herzegovina                                                                                                       ",
        "NSN_nameFrench": "Société de la Croix-Rouge de Bosnie-Herzégovine                                                                                                       ",
        "NSN_nameSpanish": "Cruz Roja de Bosnia y Herzegovina                                                                                                                     ",
        "NSN_nameArabic": "الصليب الأحمر في البوسنة والهرسك",
        "NSN_countryNameEnglish": "Bosnia and Herzegovina                                      ",
        "NSN_countryNameFrench": "Bosnie-Herzégovine                                          ",
        "NSN_countryNameSpanish": "Bosnia y Herzegovina                                        ",
        "NSN_countryNameArabic": " البوسنة والهرسك",
        "NSN_nameRussian": null,
        "NSN_countryNameRussian": null,
        "ADD_Instagram": "https://www.instagram.com/rcsbh/",
        "ADD_VolunteerWebSite": null
    },

I don't know, if it makes sense to store the data in GO (probably would be faster) or directly fetch required data from the go-api.
We should discuss this @szabozoltan69, @batpad, @frozenhelium when we are back in January.
Thanks!

**Comment by nanometrenat** [comment]
@tovari was this dealt with as part of GO country pages?

**Comment by tovari** [comment]
No, Sanjay described this as bad decision made at the system design. The fix would have high complexity and risk.
